package.json

```diff
@@ -67,7 +67,7 @@
     "lint-staged": "^15.1.0",
     "nx": "17.1.3",
     "postcss": "^8.4.5",
-    "prettier": "^2.6.2",
+    "prettier": "3.1.1",
```

Comment:
I used `--save-exact` here as recommended by Prettier docs.

Reply:
The project is using Prettier with nx.
Changes might be considered but that's out of the scope of this PR.
Please roll back on any change not related to control flow syntax.

I'll start the review after.

Reply:
as there are dedicated commits, that's fine with me then. Starting the review right now!